lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -614,6 +614,36 @@ serve as complete descriptions of the runtime of a tool.
       <xs:element name="requirement" type="Requirement" minOccurs="0" maxOccurs="unbounded"/>
       <xs:element name="container" type="Container" minOccurs="0" maxOccurs="unbounded"/>
     </xs:sequence>
+    <xs:attribute name="cores_min" type="xs:long" use="optional">
```

Comment:
Looks like xs:long isn't a valid type, but the CWL 1.2 spec allows this to be floats, so maybe we should go with
```suggestion
    <xs:attribute name="cores_min" type="float" use="optional">
```
?